coderd/database/queries/presets.sql

```diff
@@ -49,6 +49,15 @@ FROM
 WHERE
 	template_version_presets.template_version_id = @template_version_id;
 
+-- name: GetPresetParametersByPresetID :many
+SELECT
+	template_version_preset_parameters.*
+FROM
+	template_version_preset_parameters
+	INNER JOIN template_version_presets ON template_version_preset_parameters.template_version_preset_id = template_version_presets.id
```

Comment:
Do we need this `INNER JOIN`?
Looks like we can just write:
```SQL
WHERE template_version_preset_parameters.template_version_preset_id = @preset_id
```

and maybe let's use `tvpp` alias?